README.md

```diff
@@ -5,27 +5,54 @@ This repository provides a template for creating **feature specifications** for
 
 When a new feature or product is to be developed, a new pull request is created, and contributors are invited to discuss its content and propose suggestions. The goal of a feature specification is to define the expected behavior on a high level and point out corner cases that need to be addressed.
 
+The person in charge of the RP (the owner) is the person assigned to the RP. This allows for the possibility of changing it over time. To change the person assigned to the PR, this should be seen with at least the lest and the new assignee.
+
+#### Start
+
+To start a new specification, it is recommended to write the first draft by filling in only the `Summary` and `Motivation` parts. 
+
+The name of the spec must follow the pattern: `PR_number-feature-name.md`. For example, if PR number 12 is about facetting, its spec will be named `0012-facetting.md` or `0012-facetted-search.md`. It will be logical that the names of the PRs will not follow each other. This is not a problem. The fact that the name has the number of PR simplifies the fact that there is no synchronization to choose this number. 
+
+#### Draft PR
+
+Once the file is created with the two parts, `Summary` and `Motivation` filled in. It is time to create a draft PR. The PR should remain in draft until the content is, according to the author, ready for comments. However, outsiders can make some recommendations, but these are more in the spirit of helping rather than suggesting changes. 
```

Comment:
Generally, I would say that a spec starts with a functional specification. When starting a spec, there should have been discussion ahead of time, and we should already have a rough idea of what we want to do, this is for example how we processed for https://github.com/meilisearch/specifications/pull/3.

When the functional specification has been agreed upon, the spec changes owner to someone in the core team who's in charge of writing the technical specification.

Reply:
I really think that the first thing that must be filled is the motivation part. Without this part, you don't know in which direction to take the PR. And the summary explains in 1 sentence the goal of the issue, so I think it's also right to have it before talking about the functional spec.

Reply:
this should be part of a pre-rfc meeting don't you think? We must not use this repo as an idea box imo. Ideally, ideas that land here should be mature enough, no?

Reply:
I think that It will be the role of the PM/me (now) to select which PR must be created. Create a PR could be limited to only some people? I am agree with you on the point that this repo should not be an idea inbox.